axum-tracing-opentelemetry/Cargo.toml

```diff
@@ -60,3 +67,8 @@ opentelemetry_sdk = { workspace = true, features = [
   "rt-tokio",
   "testing",
 ] }
+
+[features]
+default = ["dep:instrumentation-sdk-trace"]
+# to use level `info` instead of `trace` to create otel span
+tracing_level_info = ["dep:instrumentation-sdk-info"]
```

Comment:
Instead of declaring a duplicate dependencies, you could just enable features on transitive dependencies

```suggestion
[features]
# to use level `info` instead of `trace` to create otel span
tracing_level_info = ["tracing-opentelemetry-instrumentation-sdk/tracing_level_info"]
```